components/ui/card/ListItemInfoCard.tsx

```diff
@@ -0,0 +1,78 @@
+import { useColorScheme } from 'nativewind';
+import React, { ReactElement } from 'react';
+import { View, Text, TouchableOpacity } from 'react-native';
+
+import CustomImage, { ImageBase64 } from '@/components/ui/image/CustomImage';
+import { IconSize } from '@/util/IconSize';
+
+interface ListItemInfoCardProps {
+  image: ImageBase64;
+  title?: string;
+  details: string;
+  iconProps?: IconProps;
+}
+
+interface IconProps {
+  icon: ReactElement;
+  color: string;
+  darkModeColor?: string;
+  onPress?: () => void;
+}
+
+const ListItemInfoCard: React.FC<ListItemInfoCardProps> = ({
+  image,
+  title = null,
```

Comment:
why we want to have that as default as null ?

Reply:
when user has no firstname and lastname we will only show their username. IconProps is nullable because in Group members list there  is one user with dimond (icon) and other without

Reply:
yeah but when we dont specift default its undifiend and will work the same, and in that case we want to change that specialy from undifiend => null

Reply:
ok will fix didn't know that xd